### PR TITLE
AKLS Strong motion station install

### PR DESCRIPTION
**Pull request by Thomas-Benson**
- Opening new station AKLS
- EQR120 (46657) SM sensor installed at location 20
- Axial: sensor azimuth 130

### CODE REVIEW

**Review by CBurton90 — APPROVED** [review]

**Comment by Thomas-Benson** [comment]
@CBurton90 I'm currently using the location from the network PR, is this correct?

**Comment by mnaguit** [comment]
And I might have missed it in the main ticket, can you pls double check the elevation for this new site? Is it 70? Thanks @CBurton90

**Comment by CBurton90** [comment]
Details directly from the EQR120 are:

-36.853647
174.769832
69.756m

@Thomas-Benson @mnaguit

**Comment by Thomas-Benson** [comment]
@CBurton90 @mnaguit coordinates have been updated

**Review by CBurton90 — DISMISSED** [review]

**Comment by salichon** [comment]
hi  @Thomas-Benson when a new station starts always good to noice the platform team and Mark for instance. 
This helps downstream : )

File: network/sites.csv
Comment (salichon) [inline network/sites.csv:1]: I dont think we need an elevation  to the millimeters :) 
70  should do  - 70  meters in auckland is correct ? 
(and often lat./lon with 4/5 digits is good enough i reckon )

File: network/stations.csv
Comment (salichon) [inline network/stations.csv:596]: Same issue there  - 70  meters is good when verified 
Provided that fromer AKUS is 50 meters

**Review by salichon — CHANGES_REQUESTED** [review]

File: network/stations.csv
Comment (Thomas-Benson) [reply network/stations.csv]: @salichon coordinates updated

**Review by CBurton90 — DISMISSED** [review]

**Review by mnaguit — APPROVED** [review]

**Comment by mnaguit** [comment]
Hi @Thomas-Benson . What can we do to move this PR along? 😄 Thanks in advance!

**Comment by Thomas-Benson** [comment]
Sorry, I was wasn't sure i I needed to wait for Mark's review and left this hanging. Needs another review after resolving merge conflicts.

**Comment by mnaguit** [comment]
No worries at all @Thomas-Benson , thanks for the update. 
Hi @CBurton90 @ozym , can we have another round of review for this PR? Thanks!

**Comment by ozym** [comment]
You need to fix the requested changes first.

File: network/sites.csv
Comment (Thomas-Benson) [reply network/sites.csv:1]: lat/lon has been updated to 5 sig. figures, and elevation is 70m.

**Comment by Thomas-Benson** [comment]
@ozym requested changes had been addressed, marked as resolved now

**Comment by ozym** [comment]
Cool, next step is to rebase the PR then dismiss the requested changes (won't merge without this), I'll review as well (but I probably don't need to if others have).

**Review by ozym — APPROVED** [review]

**Review by mnaguit — APPROVED** [review]

**Comment by ozym** [comment]
It would be easier to fix these conflicts if the PR wasn't on a fork.

**Review by CBurton90 — APPROVED** [review]

**Pull request merged**
